tests/clients/explorer/test_explorer.py

```diff
@@ -0,0 +1,82 @@
+from os import environ
+from random import randint, choice
+
+from jumpscale.loader import j
+from tests.base_tests import BaseTests
+
+
+class Explorer(BaseTests):
+
+    tname = environ.get("TNAME")
+    email = environ.get("EMAIL")
+    words = environ.get("WORDS")
+    explorer_url = "https://explorer.testnet.grid.tf/api/v1"
+    MYID_NAME = "identity_{}".format(randint(1, 1000))
+    explorer = j.core.identity.me.explorer
+
+    @classmethod
+    def setUpClass(cls):
+        if not all([cls.tname, cls.email, cls.words]):
+            raise Exception("Please add (TNAME, EMAIL, WORDS) of your 3bot identity as environment variables")
+        cls.me = None
+        if j.core.identity.list_all() and hasattr(j.core.identity, "me"):
+            cls.me = j.core.identity.me
+        myid = j.core.identity.new(
+            cls.MYID_NAME, tname=cls.tname, email=cls.email, words=cls.words, explorer_url=cls.explorer_url
+        )
+        myid.register()
+        myid.set_default()
+        myid.save()
+
+    @classmethod
+    def tearDownClass(cls):
+        j.core.identity.delete(cls.MYID_NAME)
+        if cls.me:
+            cls.me.set_default()
+
+    def test01_check_nodes(self):
+        """
+        #. Check the node lists in explorer client.
```

Comment:
please write description like that:
```suggestion
        """Test description.
        **Test Scenario**
        - Get a github client.
        - Create a repository.
        - Check that this repository has been created.
        """
```